markdown/org/docs/patterns/simon/instructions/en.md

```diff
@@ -461,9 +461,9 @@ Spread your shirt open so that both fronts and back lie flat with their good sid
 
 Now place your sleeve on top with the good side down, matching the top of the sleeve with the shoulder seam.
 
-> Be careful, your sleeve is not symmetric, so be sure to have the double notch (and cuff guard) on the back side, and the single notch on the front side.
+> Be careful, your sleeve is not symmetric, so be sure to have the ⊗ notch (and cuff guard) on the back side, and the ⊙ notch on the front side.
```

Comment:
Can we do:
```
have the back notch (⊗) ... and the front notch (⊙) on the ...
```
so that we have both the words and the symbols?

Reply:
Done.